### ISSUE TITLE
Update view / change dependents page

### ISSUE THREAD

**Issue by aprocik1** [department-of-veterans-affairs/va.gov-team#80538]

**Comment by katherine-fung** [comment]
Hi @aprocik1, 

I've drafted changes to this page for your review: https://preview-prod.vfs.va.gov/preview?nodeId=6663

Thanks!

**Comment by aprocik1** [comment]
Thanks @katherine-fung! I made these edits:

- I removed "find out if you’re eligible" from the intro, since we only present eligibility information for disability compensation. Ideally, I think we should add eligibility information for pension and survivors' benefits. Do you know if there's eligibility requirements for adding dependents to those benefits?
- I updated the header structure. What do you think about this edit?
- I updated the accredited representative content and link.

**Comment by katherine-fung** [comment]
Thanks for these edits, @aprocik1!! I'll ask the product team to check with SMEs if there are eligibility requirements to add dependents to pension and survivors' benefits. I can't tell from searching VA.gov (it doesn't look like there are any eligibility requirements from what I can tell - just that you need to have dependents?)

**Comment by aprocik1** [comment]
Thanks, @katherine-fung. I made some edits to try to standardize the design for each H3. What do you think?

> [H2] Can I get higher benefit payments for one or more of my dependents?
> 
> If you and your dependents meet the requirements here, you may be eligible for higher disability compensation, pension, or DIC payments.
> 
> [H3] For disability compensation
> 
> You may be eligible for higher disability compensation payments if both of these are true for you:
> 
> - You're eligible for VA disability compensation, **and**
> - You and your dependent child, spouse, or parent have a combined disability rating of at least 30%
> 
> [H3] For pension benefits
> 
> You may be eligible for higher pension payments if one of these are true for you:
> 
> - Your dependent child's income and assets are within the limits to be eligible for pension benefits, **or**
> - You and your dependent spouse's combined income and assets are within the limits to be eligible for pension benefits
> 
> **Note:** You'll report your dependent child or spouse's income and assets when you apply for pension benefits.
> 
> [Learn about the income and assets for pension benefits](https://www.va.gov/pension/veterans-pension-rates/)
> 
> [H3] For DIC benefits
> 
> You may be eligible for higher DIC payments if both of these are true for you:
> 
> - You're the surviving spouse of a Veteran, **and**
> - You have one or more children under the age of 18
> 
> **Note:** Surviving children and parents of Veterans may also be eligible for separate DIC payments.
> 
> [Learn more about VA DIC rates for spouses and children](https://www.va.gov/disability/survivor-dic-rates/)
> 
> [Learn more about VA DIC rates for parents](https://www.va.gov/disability/parent-dic-rates/)

**Comment by katherine-fung** [comment]
Thanks @aprocik1! These are great edits. I just have one change for "Note: You'll report your dependent child or spouse's income and assets when you apply for pension benefits." 

I would suggest "**Note:** You must report you and your spouse's combined income and assets when you apply for pension benefits."

When married Veterans apply for pension benefits, they always have to report their spouse's income together with their own, even if their spouse's income + assets are high. If their combined income + assets is beyond the limit, the Veteran can't get pension. That's what I meant by the original note. The Veteran doesn't have a choice between reporting it if their spouse's income/assets are low, or withholding the information if their spouse's income/assets are high. 

This is different from adding a child dependent. If a child's income/assets are too high, VA simply won't count them as a dependent. But reporting the child's income/assets isn't required, and won't disqualify the Veteran from benefits the way a spouse's income/assets might.

**Comment by katherine-fung** [comment]
I've sent screenshots of the drafted updates to the SMEs via MS Teams for review. cc @aprocik1

**Comment by ajialeilani** [comment]
Hi @katherine-fung @aprocik1 - I spoke to one of our stakeholders, Brandi Traylor, today, and she says she'll take a look at the updates. 

One note from her initial scan: 

> You and your dependent child, spouse, or parent have a combined disability rating of at least 30%

This is not accurate. The "combined disability rating" refers only to the applying Veteran's service-connected conditions. If a dependent has their own disability rating, they'll have their own award and their rating will not impact the applying Veteran's eligibility. 

I'll pass along anything else I hear from her. Thanks!

**Comment by ajialeilani** [comment]
Hi @strelichl apologies for the delay. 

In addition to the [feedback above](https://github.com/department-of-veterans-affairs/va.gov-team/issues/80538#issuecomment-2187222608), the only feedback we received was around making sure the requirements for permanently disabled children are clear: 

> It isn't clear that the medical documentation for claiming helpless child needs to show the child became helpless prior to the age of 18. The page does state the child had to be considered helpless before 18,however, it isn't clear (to me) that the Veteran has the burden of proof to provide documentation that explicitly proves the child became helpless prior to the age 18.
> 
> **X.ii.6.B.1.c. Information Necessary to Establish the Extent of the Child's Disability**
>  The information necessary to establish the extent of the child's disability includes
> the extent to which the child is and was, prior to reaching their 18th birthday, physically or mentally deficient, as evidenced by factors such as their ability to perform
> self-care functions, and
> ordinary tasks expected of a child of that age
> whether or not the child attended school and, if so, the maximum grade attended
> if any material improvement in the child’s condition has occurred
> if the child has ever been employed and, if so, the
> nature and dates of such employment, and
> amount of pay received
> whether or not the child has ever married, and
> a description of the child’s present condition.
> 
> **X.ii.6.B.1.d. Reporting Facts Based on Personal Observation**
>  The persons making statements pertaining to the child’s permanent incapacity for self-support should give detailed information of the facts known to them personally in regard to the child’s
> mental and physical condition when the child attained the age of 18 years, and
> present condition.
> Important: All statements should indicate the basis of the knowledge of the facts reported and include the name and signature of the person making the statement.

**Comment by katherine-fung** [comment]
Hi @ajialeilani! 

Can you ask the SME if the changes circled in yellow adequately address her feedback? 

![Image](https://github.com/user-attachments/assets/808fb5e8-9df8-4966-895c-6f87bdfd13c1)

**Comment by ajialeilani** [comment]
Hi @katherine-fung - I'll pass these screenshots along to her today. Sorry for the delay. 

Also, Mikki Northuis [pointed out today](https://dsva.slack.com/archives/C0547Q0K0LF/p1726762520868349) that there may be a redundancy in having a login widget on both the static page and the 686c intro page. I'm not sure if it's in scope for these updates to change the login widget for 686c to an action link. 

cc: @strelichl

**Comment by ajialeilani** [comment]
@katherine-fung we got the thumbs up from our SME!

**Comment by katherine-fung** [comment]
Great, thanks @ajialeilani !! 

These changes are in draft, and are ready to publish when the updated form is live. 

@steele-lm @strelichl can you be in touch with each other about when 674/686c updates go live, so @strelichl can assign someone from CAIA to publish the page? 

I'm moving this ticket to validate so @NaomiPMC can track and close it. cc @aprocik1

**Comment by RLHecht** [comment]
@ajialeilani We wanted to move forward with this super old ticket! I wanted to confirm from the comments above that we should remove the sign-in widget from this static page and put an action link instead. And the other changes that were made to this page have all been approved by SMEs to publish. Here's a preview link if you need to review again: https://preview-prod.vfs.va.gov/preview?nodeId=6663 If all this sounds right, we'll move forward with publishing.

**Comment by RLHecht** [comment]
Thanks @steele-lm. So does that mean the changes we have queued up for the static page may not be applicable to all users just yet? So we would want to hold on publishing until the form updates are at 100%?

**Comment by steele-lm** [comment]
@RLHecht The link you shared to view the changes is not working for me, but our previous SME did approve the content changes. I don't recall why the release of the static page updates was tied to the form updates we're making. The form updates do not impact how the benefit functions, so I think static page updates can be made independently, but I'd want @fiorella-io and @jacobworrell to weigh in. For the log in widget, it does look like the guidance was to remove the second widget from the [static page](https://www.va.gov/view-change-dependents/) and replace it with an action link to allow users to view the form intro page without logging in.

**Comment by marvolpe** [comment]
@steele-lm , @RLHecht has assigned this ticket to Amy and I.  I caught up with the ticket notes and will be ready to work on it once @fiorella-io is ready.

**Comment by steele-lm** [comment]
I created a ticket for our team to do some initial exploration on how benefit requirements for dependents are communicated: https://github.com/department-of-veterans-affairs/va.gov-team/issues/113355

**Comment by marvolpe** [comment]
@davidakennedy per our brief conversation during today's Design/Research Review:
This ticket the other old ticket I mentioned that was assigned to me. I believe it's also related to our current project, but I'm unsure whether it depends on the completion of the online form or if I can proceed independently. 
This is also the one Fiorella mentioned that Laura is aware of it, and apparently has already created a new ticket for it- see above.
Could you please help clarify the timeframe and dependencies, or let me know if should go ahead and begin work?
Thanks!

**Comment by jacobworrell** [comment]
@strelichl the V2 version of the 686c is currently in phased rollout (right now at 1% of traffic). Right now, I predict we'll be switch to 100% on  7/28, which I think is when we'd want to push the new static page content. If we run into big issues, that day could change, but I wanted to give you a heads ups.

**Comment by bethpottsVADEPO** [comment]
@jacobworrell My team may have a more up-to-date ticket they're working in. @mistymg can you look into this?

**Comment by mistymg** [comment]
@marvolpe @va-agw You all can put this on hold for now. @bethpottsVADEPO has noted that there are a lot of old drafts in Drupal for this page that were never published. She's going to check on if any of those drafts can be published. This will just help us determine what actually need to be updated on this page.
